docs/conf.py

```diff
@@ -110,6 +129,29 @@
 html_static_path = ['_static']
 
 
+# -- Custom build operations -------------------------------------------------
+def metadata_to_rst(app):
```

Comment:
Okay this is way cleaner. And plus now I know how to inject behavior into Sphinx!  I think we'll probably want to do a lot more exporting of metadata and other information programmatically into the docs going forward, so this is good.

It also lets us cleanup after the build is finished, rather than needing to have a line in Tox that does it.